Paranormal/Paranormal/ToolSettings.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+public class ToolSettings {
+    weak var document : Document?
+
+    init(){
+    }
+
+    public var colorAsAngles : Array<Float> = [0,0] {
+        didSet {
+            updateToolSettingsView()
+        }
+    }
+
+    public func updateToolSettingsView() {
+        let panelVC = document?.singleWindowController?.panelsViewController
+        panelVC?.toolSettingsViewController?.updateSettingsSliders()
+    }
+
+    public var colorAsNSColor : NSColor {
+        let dir = colorAsAngles[0] * Float(M_PI) / 180.0
+        let pitt = colorAsAngles[1] * Float(M_PI) / 180.0
+        let x = sin(dir) * sin(pitt) * 0.5 + 0.5
+        let y = cos(dir) * sin(pitt) * 0.5 + 0.5
+        let z = cos(pitt) * 0.5 + 0.5
+        println(x)
+        println(y)
+        println(z)
+        return NSColor(red: CGFloat(x),
+            green: CGFloat(y), blue: CGFloat(z), alpha: 1.0)
+    }
+
+    public func setColorAsNSColor(color : NSColor) {
+        let normalZero : Float = 127.0 / 255.0;
+        var x = 2.0 * (Float(color.redComponent) - normalZero)
+        var y = 2.0 * (Float(color.greenComponent) - normalZero)
+        var z = 2.0 * (Float(color.blueComponent) - normalZero)
+        [x,y,z].map {(n) -> Float in if (abs(n) < 0.000000001) {return 0.000000001}; return n}
+
+        var degree = atan2(x, y)*180.0/Float(M_PI)
+        var pitch = atan2(sqrt(x*x + y*y),z)*180.0/Float(M_PI)
+        degree = ((degree > 0) ? degree : (degree+360))
+        colorAsAngles = [round(degree*1e6)/1e6, round(pitch*1e6)/1e6]
+    }
+
+    public var size : Float = 30.0 {
+        didSet {
+            updateToolSettingsView()
+        }
+    }
+    public var strength : Float = 1.0 {
+        didSet {
+            updateToolSettingsView()
+        }
+    }
+    public var hardness : Float = 0.9 {
+        didSet {
+            updateToolSettingsView()
+        }
+    }
+    public var gaussianRadius : Float = 30 {
```

Comment:
I know we haven't been doing this, but a few comments here would be nice. Especially considering we are releasing. It is hard to know that the gaussianRadius is for the smooth brush.